README.md

```diff
@@ -81,10 +81,10 @@ Please refer to the individual package documentation for further details.
 
 | Kubernetes Version |   Compatibility    | Notes           |
 | ------------------ | :----------------: | --------------- |
-| `1.28.x`           | :white_check_mark: | No known issues |
 | `1.29.x`           | :white_check_mark: | No known issues |
 | `1.30.x`           | :white_check_mark: | No known issues |
 | `1.31.x`           | :white_check_mark: | No known issues |
+| `1.32.x`           | :white_check_mark: | No known issues |
```

Comment:
let's mark 1.32 as tech preview like we did on networking (https://github.com/sighupio/fury-kubernetes-networking/pull/90)
```suggestion
| `1.32.x`           | :white_check_mark: | Tech preview |
```